### DISCUSSION TITLE
v2.11.8

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1525] [Announcements]
## What's Changed
* Fix updater to only treat `branch` as `nightly` if it's not `main`

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v2.11.7...v2.11.8

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v2.11.8'>v2.11.8</a>.</em>

**Comment by Kudray2** [comment]
Greetings!
I'm really inpressed by Astronvim performance and usability.
But how to make it auto update imports after file rename/move in TS projects?
